File: src/test/java/br/com/model/entities/classes/telefone/NovoTelefoneTest.java
package br.com.model.entities.classes.telefone;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.entities.classes.telefone.NovoTelefone;
import br.com.newObjects.model.entities.classes.usuario.NovoCliente;
import br.com.newObjects.model.entities.classes.usuario.NovoFornecedor;

public class NovoTelefoneTest {
    @Test
    void testNovaListaTelefoneCliente() {
        // cliente
        System.out.println(NovoTelefone.novaListaTelefoneCliente(NovoCliente.novoCliente()).toString());

    }

    @Test
    void testNovaListaTelefoneFornecedor() {
        // fornecedor
        System.out.println(NovoTelefone.novaListaTelefoneFornecedor(NovoFornecedor.novoFornecedor()));
    }
}

File: src/test/java/br/com/model/NovoTelefoneTest.kt
package br.com.model

import br.com.newObjects.model.usuario.NovoFornecedor
import org.junit.jupiter.api.Test

class NovoTelefoneTest {
    @Test
    fun testNovaListaTelefone() {
        val fornecedor = NovoFornecedor.novoFornecedor()
        println(fornecedor!!.telefones)
    }
}
